xskillscore/tests/test_deterministic.py

```diff
@@ -307,3 +307,16 @@ def test_dim_empty_list(a, b, metrics):
         metric, _metric = metrics
         res = metric(a, b, dim=[])
         assert len(res.dims) == len(a.dims), print(res.dims)
+
+
+@pytest.mark.parametrize('metrics', correlation_metrics)
+def test_correlation_broadcasts(a, b, metrics):
```

Comment:
Why just correlation? Can we also add the distance metrics here or is that just overkill?

Reply:
Distance metrics already worked before but yes could include